exes/mqtt-broadcaster/inc/config.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <tfc/confman.hpp>
+#include <tfc/confman/observable.hpp>
+
+// File under /etc/tfc/mqtt-broadcaster/def/mqtt_broadcaster.json which lists the signals that are supposed to be published
+// on the mqtt broker. Updating the signals in this file will cause the running program to stop and restart
+struct config {
+  std::string address{};
+  std::string port{};
```

Comment:
why do we have port as string here instead of uint16_t?

Reply:
Because the code also supports port name resolution, f.e. "mqtt" resolved to "1883". Another possible name would be "port_service" or something similar. Change the name?

Reply:
`mqtts` is also possible for port 8883.